collections/_drafts/agent-forwarding.md

```diff
@@ -1,15 +1,16 @@
 ---
 title: OpenINF Next-Gen Guidance on Agent Forwarding
 category: contributing
-authors: @OpenINF/wg-a-team
-published: false
-permalink:
-layout:
+permalink: /docs/dev/internals/contributing/agent-forwarding/
 relevant_urls:
-- https://github.com/OpenINF/docker-fisher/issues/5 # psusan + auth awkwardness
-- https://dev.gnupg.org/T3883 # Win32-OpenSSH support for gpg-agent's ssh-agent
+  - https://github.com/OpenINF/docker-fisher/issues/5 # psusan + auth awkwardness
+  - https://dev.gnupg.org/T3883 # Win32-OpenSSH support for gpg-agent's ssh-agent
 ---
 
```

Comment:
```suggestion

${toc}

```

i am looking at:

🔗 https://github.com/nagaozen/markdown-it-toc-done-right?tab=readme-ov-file#tldr

Reply:
theoretically, the plugin adds parser support for the `markdownlint-lint` linter, but we do not current use markdown-it plugins to build the markdown, so i do not expect the any result except for the parser not to trip up on this

Reply:
i think we need a better way for ToCs to be generated (preferably a jekyll plugin at this point), but something i want to point out is that jekyll has rendering phases:

- https://jekyllrb.com/docs/rendering-process/

Reply:
okay, some options:

🔗 https://github.com/allejo/jekyll-toc

i am thinking we will be going w/ this solution and will rm toc-done-right in a separate pr